SUSE/rmt
# Removing danger from RMT repository

We moved to Github actions which introduced issues with danger.
Thats why we crafted our own checks which work with the actions.

**How to review this pull request**

- Check that the pipeline is green for this pull request
- Check there is no more danger left in RMT
